pmd-java/src/main/resources/category/java/design.xml

```diff
@@ -28,7 +28,7 @@ protected constructor in order to prevent instantiation than make the class misl
 <![CDATA[
 //ClassOrInterfaceDeclaration
     [@Abstract = 'true']
-    [count(//MethodDeclaration) + count(//ConstructorDeclaration) = 0]
+    [count(.//MethodDeclaration) + count(.//ConstructorDeclaration) = 0]
```

Comment:
This actually fixes some false-negative when nested classes are in use...
See e.g. https://chunk.io/pmd/f0f07b25b93b453e993fa6b0d71b44e6/diff/checkstyle/index.html#A1

Reply:
This still counts methods of classes nested inside the class under scrutiny. I'll fix this and add those as test cases